frontend/src/concepts/pipelines/content/configurePipelinesServer/ObjectStorageSection.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  FormSection,
+  Title,
+  Radio,
+  FormGroup,
+  Select,
+  SelectOption,
+  Text,
+  TextInput,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import React from 'react';
+import AWSField from '~/pages/projects/dataConnections/AWSField';
+import useDataConnections from '~/pages/projects/screens/detail/data-connections/useDataConnections';
+import { getDataConnectionDisplayName } from '~/pages/projects/screens/detail/data-connections/utils';
+import { DataConnection } from '~/pages/projects/types';
+import { useContextResourceData } from '~/utilities/useContextResourceData';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { EMPTY_AWS_SECRET_DATA } from '~/pages/projects/dataConnections/const';
+import { PipelineServerConfigType } from './ConfigurePipelinesServerModal';
+
+const DISABLED_FOLDER_PATH = (
+  <FormGroup
+    label="Folder path"
+    helperText="/metadata and /artifacts folders are automatically created in the default root folder"
+  >
+    <TextInput isDisabled aria-label="disabled folder path field" value="/" />
+  </FormGroup>
+);
+
+type ObjectStorageSectionProps = {
+  setConfig: (config: PipelineServerConfigType) => void;
+  config: PipelineServerConfigType;
+};
+
+export const ObjectStorageSection = ({ setConfig, config }: ObjectStorageSectionProps) => {
+  const { namespace } = usePipelinesAPI();
+  const dataConnections = useContextResourceData<DataConnection>(useDataConnections(namespace));
+  const [existingDataConnectionOpen, setExistingDataConnectionOpen] = React.useState(false);
+  return (
+    <FormSection
+      title={
+        <>
+          <Title headingLevel="h2">Object storage connection</Title>
+          <Text component="p" className="form-subtitle-text">
```

Comment:
CSS files should be imported WITH the css usage of the classname. Otherwise you end up running the risk of the file not being included if some other component gets deleted or repurposed.

Reply:
oh i see i am importing it elsewhere but it gets covered here.... so long as the other component is used somewhere. better to put it in both

++